api/authorization.yml

```diff
@@ -1133,14 +1133,14 @@ paths:
           default:
             $ref: "#/components/responses/ServerError"
 
-  /auth/users/{userId}/external/principals/{principalId}:
+  /auth/users/{userId}/external/principals:
```

Comment:
I think userID should also be in the query
It will be less painful in case we decide we don't really want to attach principal to users

Reply:
we do want to attach, it operates on a user resource.
the only reason removing the principal id is due to limitation of the api.